core/com.b2international.snowowl.core.rest.tests/src/com/b2international/snowowl/core/rest/resource/ResourceApiTest.java

```diff
@@ -174,5 +175,51 @@ public void deleteAllResources() {
 			.getSync(1, TimeUnit.MINUTES); 
 		});
 	}
+	
+	@Test
+	public void sortByResourceTypeAsc() {
+		final String id1 = "A";
+		final String id2 = "B";
+		final String id3 = "C";
+		final String id4 = "D";
+		
+		
+		createCodeSystemWithStatus(id1, "active");
+		createCodeSystemWithStatus(id2, "active");
+		
+		BundleApiAssert.createBundle(BundleApiAssert.prepareBundleCreateRequestBody(id3));
+		BundleApiAssert.createBundle(BundleApiAssert.prepareBundleCreateRequestBody(id4));
+		
+		assertResourceSearch(ImmutableMap.of("sort", ImmutableList.of("typeRank:asc", "title:asc")))
+			.statusCode(200)
+			.body("total", equalTo(4))
+			.body("items[0].id", equalTo(id3))
+			.body("items[1].id", equalTo(id4))
+			.body("items[2].id", equalTo(id1))
+			.body("items[3].id", equalTo(id2));
```

Comment:
No need to change this now, but `org.hamcrest.Matchers#contains` asserts that instances of `Iterable` contain the specified elements in the same order and numbers:

```java
.body(items, contains(id3, id4, id1, id2)); // "items" contains exactly 4 elements, in the given order
```

Reply:
The items are resources and we only have the ids. Then I need to get the resources to match.

Reply:
Correct. REST Assured uses groovy's GPath syntax for selecting parts of the response body, which allows navigating across array elements, so the first argument should be changed to `items.id` in that case.

- https://github.com/rest-assured/rest-assured/wiki/Usage#example-1---json
- http://groovy-lang.org/processing-xml.html#_gpath

Reply:
Done